Makefile

```
@@ -117,8 +97,52 @@ run-test-db:
   ./scripts/neo4j-wait-for-start;
 	make init-db
 
-clean-deps: unprepublish
-	rm -rf packages/*/node_modules
-	rm -rf node_modules
-	rm package-lock.json
-	npm install
+test:
+	@if [ -z $(CI) ]; \
+		then NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema DEBUG=true TIMEOUT=500000 \
+			jest --config="./jest.config.js" "${pkg}.*__tests__.*${spec}.*.spec.js" --testEnvironment=node --watch; \
+		else NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
+			jest --config="./jest.config.js" "__tests__.*/*.spec.js" --testEnvironment=node --maxWorkers=1 --ci --reporters=default --reporters=jest-junit --detectOpenHandles --forceExit; \
```

I wonder if `--runInBand` is faster than `maxWorkers=1` as this would avoid creating a worker pool?

---

Happy to give it a go
```suggestion
			jest --config="./jest.config.js" "__tests__.*/*.spec.js" --testEnvironment=node --runInBand --ci --reporters=default --reporters=jest-junit --detectOpenHandles --forceExit; \
```